website/templates/themes.html

```diff
@@ -0,0 +1,142 @@
+{% extends "base.html" %}
+{% block page_content %}
+    <h1> Home Themes </h1>
+    <br>
+    <ul class="nav nav-pills">
+        <li role="presentation" class="dropdown active">
+            <a class="dropdown-toggle" data-toggle="dropdown" href="#" role="button" aria-haspopup="True"
+               aria-expanded="False">
+                <span class="glyphicon glyphicon glyphicon-plus" aria-hidden="true"></span> Add New Theme
+            </a>
+            <ul class="dropdown-menu">
+                <div class="input-group">
+                    <input type="text" class="form-control" placeholder="Theme Name" aria-describedby="basic-addon2">
+                    <span class="input-group-addon" id="basic-addon2"> <span
+                            class="glyphicon glyphicon-plus"></span></span>
+                </div>
+            </ul>
+        </li>
+
+        <li role="presentation" class="dropdown active">
+            <a class="dropdown-toggle" data-toggle="dropdown" href="#" role="button" aria-haspopup="True"
+               aria-expanded="False">
+                <span class="glyphicon glyphicon-minus" aria-hidden="true"></span> Remove Existing Theme
+            </a>
+            <ul class="dropdown-menu">
+                <div class="input-group">
+                    <input type="text" class="form-control" placeholder="Theme Name" aria-describedby="basic-addon2">
+                    <span class="input-group-addon" id="basic-addon2"> <span
+                            class="glyphicon glyphicon-minus"></span></span>
+                </div>
+            </ul>
+        </li>
+    </ul>
+    <br>
+    <b><p> Below is a list of all of your set themes. </p></b>
+
+    <table class="table table-striped">
+        <tr>
+            <th>Name</th>
+            <th> Status</th>
+        </tr>
+        <tr>
+            <td>
+                <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#myModal"> Weekend Away
+                </button>
+            </td>
+
+            <td>
+                <div class="btn-group btn-toggle" data-toggle="buttons">
+                    <label class="btn btn-primary active">
+                        <input type="radio" name="options" value="option1"> On
```

Comment:
Why is there the on/off option? Wouldn't it be more logical to have an "activate" button per theme and just highlight the currently active theme if there is one? What do you think?